posts/navarroaxel/18.git-rebase-interactive.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Rewriting the history with Git rebase
+published: true
+description: How to squash commits and keep your branch up-to-date using git rebase 
+tags: 'git, bash, productivity, tooling'
+cover_image: ./assets/git-rebase.jpg
+id: 1949937
+---
+
+As when we work in a new feature or bug fix it is recommended to keep our branch up-to-date with the target branch (`main`, `development`, etc.), we can use the [`git rebase`](https://git-scm.com/docs/git-rebase) command instead of `git merge` in order to keep our Git history cleaner.
```

Comment:
```suggestion
As when we work in a new feature or bug fix it is recommended to keep our branch up-to-date with the target branch (`main`, `development`, etc.), we can use the [`git rebase`](https://git-scm.com/docs/git-rebase) command instead of `git merge` in order to keep our Git history clean.
```